content/docs/started/k8s/kfctl-istio-dex.md

```diff
@@ -0,0 +1,853 @@
++++
+title = "Multi-user, auth-enabled Kubeflow with kfctl_istio_dex"
+description = "Instructions for installing Kubeflow with kfctl_istio_dex.yaml config"
+weight = 4
++++
+
+Follow these instructions if you want to install Kubeflow on an existing Kubernetes cluster.
+
+This installation of Kubeflow is geared towards existing Kubernetes
+clusters and does not depend on any cloud-specific feature.
+
+
+## Architecture overview
+
+In this reference architecture, we use [Dex](https://github.com/dexidp/dex) and
+[Istio](https://istio.io/) for vendor-neutral authentication.
+
+This deployment works well for on-prem installations, where companies/organizations need LDAP/AD integration for multi-user authentication, and they don't want to depend on any cloud-specific feature.
+
+![kfctl_istio_dex_architecture](../../kfctl_istio_dex-architecture.svg)
+
+Read the relevant [article](https://journal.arrikto.com/kubeflow-authentication-with-istio-dex-5eafdfac4782) for more info about this architecture.
+
+
+## Before you start
+
+The instructions below assume that you have an existing Kubernetes cluster.
+
+Configuring your installation with {{% config-file-istio-dex %}} has a few options you should consider:
+
+* **Disabling istio installation** - If your Kubernetes cluster
+  has an existing Istio installation you may choose to not install Istio by removing
+  the applications `istio-crds` and `istio-install` in the configuration file
+  {{% config-file-istio-dex %}}.
+
+* **Istio configuration for trustworthy JWTs** - This configuration uses Istio version
+  1.3.1 with SDS enabled, which requires you to use Kubernetes 1.13 or later. You may need
+  to add extra configurations to your Kubernetes as mentioned in
+  [Istio's blog](https://istio.io/blog/2019/trustworthy-jwt-sds/).
```

Comment:
@sarahmaddox I've added this additional note for installation. Please review it.

Reply:
Thanks, LGTM.